### ISSUE TITLE
Issue with specto 

### ISSUE THREAD

**Issue by jpix88** [mrknow/filmkodi#546]
Specto fork error 
Check the log. 
Checked everything nothing I do resolves issue?

**Comment by mrknow** [comment]
Do you have Kodi 16.1 ? Maybe the error was caused by the update process.
wait 15 minutes and try it again
If you still will this error occurred, reinstall the plug according to the instructions http://filmkodi.com/en/specto-1080p-kodi-add/

**Comment by jpix88** [comment]
Yes I was on 16.1and I have tried removing the addon and reinstalling it and the same error comes up.

**Comment by mrknow** [comment]
OK -  need logs
- install kodi log uploader https://www.youtube.com/watch?v=Fyh32fEFvfw
- paste here link to logs

**Comment by jpix88** [comment]
I have just changed to another kodi box and it's working fine. Any further issues I will let you know.  Thank you for your help.

**Comment by kylelliott** [comment]
mine was working and doesn't work

**Comment by kylelliott** [comment]
I really need to work

**Comment by kylelliott** [comment]
I just keep getting Specto fork error 
Check the log

I dont know what to do

**Comment by elderberry999** [comment]
Mine tries to install but at the end of install failed to install dependency. Do i need to have something install prior to installing this addon

**Comment by MeDeVILe** [comment]
same issue
specto 2016.10.28.2
Linux mint 18
Kodi 16.1

logs: http://pastebin.com/WSSttZnW

**Comment by mrknow** [comment]
I think, you must install repository to automatically fetch missing dependencies (including: script.specto.media, script.mrknow.urlresolver)

How to install repository http://filmkodi.com/en/specto-1080p-kodi-add/

short instruction:
- SYSTEM > File Manager -> Add source
- Add source http://kodi.filmkodi.com 
- Go to System-> Add-on's >Install from zip file
- Select repository.filmkodi.com
- Select repository.filmkodi.com-2.10.zip, and wait for addon
- Go to System-> Add-on's >Install from repository -> Video Add-on's
- install Specto

**Comment by MeDeVILe** [comment]
Yes. It's working for me. First I uninstalled all addons from mrknow than I added repository as shown above and installed addons back.

Dzięki @mrknow

**Comment by ymadi** [comment]
I installed Kodi on windows 10. I try to install specto, but it failed to install dependency. I tried to follow the instructions above (1. Uninstalled Mrknow addons, 2. Installed script.specto.media, and then script.mrknow.urlresolver, 3. Installed repository.filmkodi.com-2.10.zip, 4. Installed from repository specto..fork), and it still same result.

**Comment by mrknow** [comment]
@ymadi - I added script.module.metahandler to my repository. Please install again Specto. it should work now

**Comment by grantwar** [comment]
I'm getting the same issue today, unable to install specto due to dependency failure.
